site/en/docs/extensions/reference/runtime/index.md

```diff
@@ -1,71 +1,110 @@
-  [getPackageDirectoryEntry](/docs/extensions/reference/runtime/#method-getPackageDirectoryEntry), and
-  [getPlatformInfo](/docs/extensions/reference/runtime/#method-getPlatformInfo).
+  [getManifest][method-getmanifest], and
+  [getPlatformInfo][method-getplatforminfo].
 
 Managing extension lifecycle and options
 
-: These methods let you perform some meta-operations on the extension, and display the options page
-  to the extension user. Methods in this category include
-  [reload](/docs/extensions/reference/runtime/#method-reload),
-  [requestUpdateCheck](/docs/extensions/reference/runtime/#method-requestUpdateCheck),
-  [setUninstallURL](/docs/extensions/reference/runtime/#method-setUninstallURL), and
-  [openOptionsPage](/docs/extensions/reference/runtime/#method-openOptionsPage).
+: These properties let you perform some meta-operations on the extension, and display the options page. 
+Methods and events in this category include
+  [onInstalled][method-oninstalled],
+  [onStartup][method-onstartup],
+  [openOptionsPage][method-openoptionspage],
+  [reload][method-reload],
+  [requestUpdateCheck][method-requestupdatecheck], and
+  [setUninstallURL][method-setuninstallurl]
 
-Device restart support
+Helper utilities
+
+: These methods provide utility such as the conversion of internal resource representations to
+  external formats. Methods in this category include
+  [getURL][method-geturl].
+
+Kiosk mode utilities
 
 : These methods are available only on ChromeOS, and exist mainly to support kiosk implementations.
   Methods in this category include
-  [restart](/docs/extensions/reference/runtime/#method-restart) and
-  [restartAfterDelay](/docs/extensions/reference/runtime/#method-restartAfterDelay).
+  [restart][method-restart] and
+  [restartAfterDelay][method-restartafterdelay].
 
-Helper utilities
+## Permissions {: #perms }
 
-: These methods provide utility such as the conversion of internal resource representations to
-  external formats. Methods in this category include
-  [getURL](/docs/extensions/reference/runtime/#method-getURL).
+Most methods on the Runtime API do **not** require any permission, except for
+[sendNativeMessage][method-sendnativemessage] and [connectNative][method-connectnative], which
+require the `nativeMessaging` permission.
+
+## Manifest {: #manifest }
 
+The following example shows how to declare the `nativeMessaging` permission in the manifest:
 
-## Manifest
+{% Label %}manifest.json:{% endLabel %}
 
-Most methods on the runtime API do not require any permission to use. However,
-[sendNativeMessage](/docs/extensions/reference/runtime/#method-sendNativeMessage) and
-[connectNative](/docs/extensions/reference/runtime/#method-connectNative) require the
-`nativeMessaging` permission to be declared in your manifest.
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "nativeMessaging"
+  ],
+  ...
+}
+```
 
+## Use cases {: #examples}
 
+### Add an image to a web page {: #example-get-url }
```

Comment:
I changed this title to make it more searchable. Thoughts?

Reply:
No objections here!